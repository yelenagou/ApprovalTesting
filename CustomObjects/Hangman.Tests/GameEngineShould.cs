using ApprovalTests;
using ApprovalTests.Reporters;
using Hangman;
using Xunit;

namespace Tests
{
    public class GameEngineShould
    {
        [Fact]
        [UseReporter(typeof(DiffReporter))]
        public void UpdateGameState()
        {
            var sut = new GameEngine("Pluralsight");

            sut.Guess('x');
            sut.Guess('p');
            sut.Guess('l');

            Approvals.Verify(sut);
        }
    }
}
