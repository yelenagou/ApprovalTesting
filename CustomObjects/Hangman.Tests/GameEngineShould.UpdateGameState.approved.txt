Guessed letters: XPL
Guesses remaining: 9
PL---L-----